frontend/src/metabase-types/api/collection.ts

```diff
@@ -139,10 +140,9 @@ export type ListCollectionItemsRequest = {
   models?: CollectionItemModel[];
   archived?: boolean;
   pinned_state?: "all" | "is_pinned" | "is_not_pinned";
-  sort_column?: "name" | "last_edited_at" | "last_edited_by" | "model";
-  sort_direction?: "asc" | "desc";
   namespace?: "snippets";
-} & PaginationRequest;
+} & PaginationRequest &
+  Partial<SortingOptions>;
```

Comment:
Reusing `SortingOptions` since we already use it in the collection browser, and now that it's using the typed entity loader, TS is throwing errors since `sort_column` isn't technically a `string` type